back-end/services/watchlist.py

```diff
@@ -35,34 +38,44 @@ def createWatchlists(userID, watchlistName:str):
             nullWrapper = None
             createdTime = datetime.datetime.now().timestamp()
             cursor.execute("""INSERT INTO `WATCHLISTS` ( `user_id`, `wl_name`, `created`, `updated`,`deleted`) VALUES
-            (%s, %s, %s, %s,%s)""", (userID, watchlistName,createdTime,createdTime,nullWrapper))
+            (%s, %s, %s, %s,%s)""", (user_ID, watchlistName,createdTime,createdTime,nullWrapper))
             cnx.commit()
-            ret = 200
+            cursor = cnx.cursor(buffered=True)
+            cursor.execute("""SELECT wl_id FROM WATCHLISTS WHERE user_id = %s and wl_name = %s and deleted is null""", (user_ID, watchlistName))
+            wl_id = cursor.fetchone()[0]
+            
+            ret = wl_id
+
+            if (len(tickers) > 0):
+                WatchlistService.addTickersToWatchlist(wl_id, user_ID, True, *tickers)
+                ret = wl_id
+
         except Error as e:
             if (e.errno == errorcode.ER_DUP_ENTRY):
                 ret = ("Error: Duplicate Entry",409)
             else:
-                ret = ("Error: ",500)
+                ret = ("Error: ",e,500)
```

Comment:
this is my bad - should be 400. I think you can commit the change in my comment below if you approve my review?